cypress/src/components/ProductCard.js

```diff
@@ -0,0 +1,8 @@
+/*globals cy*/
+class ProductCard {
+    clickOnProductByName(productName) {
+        cy.get('.col-lg-9').contains(productName).click()
```

Comment:
<img width="386" alt="Screenshot 2024-08-06 at 09 34 27" src="https://github.com/user-attachments/assets/bc5a24d9-52a7-4809-953e-dbd1f5d67ba1">

It's not a good practice to use class for selecting the element. Try to find the unique value of the element
https://docs.cypress.io/guides/references/best-practices